src/server/api/user.js

```diff
@@ -174,6 +174,129 @@ export async function getUsers(
   }
 }
 
+export const getUserTodos = async (
```

Comment:
this is completely pulling out the `todos` function into a new function, `getUserTodos` for a function and adding `queryFilter` as an argument, so we can use the same code for `notifications` where we pass queryFilter to be only the assignmentIds that were notifified.